### DISCUSSION TITLE
Enabling double month view

### DISCUSSION THREAD

**Discussion by t-barclay** [easepick/easepick#4] [Q&A] [answered]
How do I enable the double month view seen at the top of the overview page? I can't seem to see a demo for this anywhere

<img width="803" alt="Screenshot 2022-03-23 at 19 55 50" src="https://user-images.githubusercontent.com/32012495/159784812-4f3a00fb-76a7-440e-b72c-015cab8a0ac4.png">

**Comment by wakirin** [comment, marked as answer]
Use `grid` and `calendars` options.
https://easepick.com/packages/core#options

https://jsfiddle.net/waki/owdp8ygt/
